website/templates/konfig-credits.php

```diff
@@ -124,3 +124,10 @@
     'link' => 'https://www.google.com/recaptcha/',
     'desc' => 'Secure captcha (stop bots)',
 ];
+$config['gk_credits'][] = [
```

Comment:
if you target new-theme, then new related file is `website\app\GeoKrety\Controller\HallOfFame.php`